server/index.js

```diff
@@ -40,7 +40,7 @@ app.get('/courses/:courseId', wrap(async (req, res) => {
 }));
 
 app.post('/courses', wrap(async (req, res) => {
-  // expecting course: { name, description, steps }
+  // expecting course: { name, description, url, steps }
```

Comment:
You saw this "url" addition in my code but it was red herring. The urls are actually stored in the steps array. This should just be expecting {name, description, steps}.

Reply:
dah! right u are